deployments/helm-chart/templates/controller-appprotect-dos-arb.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.controller.appprotectdos.enable }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "nginx-ingress.appProtectDosArbName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "nginx-ingress.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ include "nginx-ingress.appProtectDosArbName" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "nginx-ingress.appProtectDosArbName" . }}
+    spec:
+      containers:
+        - name: {{ include "nginx-ingress.appProtectDosArbName" . }}
```

Comment:
does the arbitrator support any cli arguments? for example, log verbosity. If it supports something that the admin might want to configure, we should expose them through the helm chart parameters.

Reply:
No cli args